docs: add the tut for amazon aurora (#837)
bytebase/bytebase.com: content/blog/database-change-management-with-amazon-aurora.md

  bytebase/bytebase:%%bb_version%% \
  --data /var/opt/bytebase \
  --port 8080
````

2. Bytebase is now running via Docker, and you can access it via `localhost:5678`.
![docker](/static/blog/database-change-management-with-amazon-aurora/docker.webp)

1. Visit `localhost:5678` in your browser. Register the first admin account which will be granted [`Workspace Owner`](/docs/concepts/roles-and-permissions).
![bb-register](/static/blog/database-change-management-with-amazon-aurora/bb-register.webp)

## Step 2 - Add an Amazon Aurora Instance to Bytebase

In Bytebase, ​​an Instance could be your on-premises MySQL instance, an AWS RDS instance etc, in this tutorial, ​an **Instance** is your `Amazon Aurora MySQL instance`.

1. Visit `localhost:5678` and log in as `Workspace Owner`.
![bb-login](/static/blog/database-change-management-with-amazon-aurora/bb-login.webp)

2. Click **Add Instance**.
![bb-home-add-instance](/static/blog/database-change-management-with-amazon-aurora/bb-home-add-instance.webp)

3. Fill in the fields and click **Create**. Pay attention to these fields:

**Type**: `MySQL`, if you use Aurora PostgreSQL, choose `PostgreSQL` instead.

> **Environment**: choose `Test`, if you choose `Prod`, you'll need manual approval for all future change requests by default, let's keep it simple for this tutorial. However, it's all configurable.

Reviewer: ```suggestion
**Environment**: choose `Test`, if you choose `Prod`, you'll need manual approval for all future change requests by default, let's keep it simple for this tutorial.
```

![bb-projects-new-project](/static/blog/database-change-management-with-amazon-aurora/bb-projects-new-project.webp)

## Step 4 - Create an Amazon Aurora Database via Bytebase

In Bytebase, a **Database** is created by `CREATE DATABASE xxx`. A database always belongs to a single **Project**. An **Issue** represents a specific collaboration activity between Developer and DBA for when creating a database, altering a schema. It's similar to the issue concept in other issue management tools.

1. Click **Projects** > `TestAurora` on the left sidebar. Click **New DB** to create a new database. You can transfer your existing 
   
2. Fill the form with **Name** - `db_demo`, **Environment** - `Test`, and **Instance** - `Amazon Aurora MySQL`. Click **Create**.
   
3. Bytebase will create an issue to create the database automatically. As it's the `Test` environment, the issue will run without waiting for your approval by default. Click **Resolve**, and the issue is `Done`.
![bb-issue-create-dbdemo](/static/blog/database-change-management-with-amazon-aurora/bb-issue-create-dbdemo.webp)

## Step 5 - Create a Table in Amazon Aurora MySQL

> In Step 4, you actually created an issue using UI workflow and then executed it. Let’s make it more explicit.

Reviewer: ```suggestion
In Step 4, you created an issue to create a database using UI workflow and then executed it. Let’s continue to create a table.
```

1. Visit your project, and click on **Alter Schema**.

2. Choose `db_demo`  and click **Next**.
![bb-alter-schema-select-db](/static/blog/database-change-management-with-amazon-aurora/bb-alter-schema-select-db.webp)

3. This is where you get to try out the **Schema Editor**. It’s a visual editor for schema changes. Create a table called `t1` with 2 columns: `id` and `name`.
![bb-schema-editor](/static/blog/database-change-management-with-amazon-aurora/bb-schema-editor.webp)

4. Click **Preview issue**, and Bytebase will automatically preview an issue with the corresponding SQL statement. Verify it's right, and click **Create**.
![bb-issue-create-t1](/static/blog/database-change-management-with-amazon-aurora/bb-issue-create-t1.webp)

> 5. Since it’s for the `Test` environment, the issue is automatically approved by default. However, there are several tasks auto running before the SQL execution. For SQL review, You may [customize your own SQL Review policies](/docs/sql-review/review-policy/overview).

Reviewer: ```suggestion
5. The issue is automatically approved by default since it’s for the `Test` environment. Meanwhile, Bytebase has  run several task checks before executing the SQL, and one such task check is called SQL Reivew. You may [customize your own SQL Review policies](/docs/sql-review/review-policy/overview).
```

![bb-task-checks](/static/blog/database-change-management-with-amazon-aurora/bb-task-checks.webp)

6. Click **Resolve issue**. The issue will become `Done` .
![bb-issue-t1-done](/static/blog/database-change-management-with-amazon-aurora/bb-issue-t1-done.webp)

7. From the issue page, click **View change**, and you can see schema diff.
![bb-change-diff-t1](/static/blog/database-change-management-with-amazon-aurora/bb-change-diff-t1.webp)

## Step 6 - Add Some Data and Query

> Bytebase support [Rollback for MySQL](https://www.bytebase.com/docs/change-database/rollback-data-changes).

Reviewer: This should be put in the next section

1. Go to the project `TestAurora` , and click **Change Data**.
2. Choose `db_demo`  and click **Next**.
3. Fill in the SQL as follows and then click **Create**.

```sql
INSERT INTO
  t1
VALUES
  (1, 'Adela');
```

4. After its execution, Click **Resolve**.
![bb-issue-insert-data-done](/static/blog/database-change-management-with-amazon-aurora/bb-issue-insert-data-done.webp)

5. Click **SQL Editor** on the left side bar. Input the query and click **Run**. You can see the new row is there.
![bb-sql-editor-select-1](/static/blog/database-change-management-with-amazon-aurora/bb-sql-editor-select-1.webp)

# Step 7 - Rollback the Data Change

After a data change completes, Bytebase can parse MySQL binary logs and build rollback SQL statements from the logs. This allows you to revert that data change if needed.

1. Go back to the issue, turn the **SQL Rollback** on.
![bb-issue-before-rollback](/static/blog/database-change-management-with-amazon-aurora/bb-issue-before-rollback.webp)

> 2. It failed because you lack some configuration.

Reviewer: ```suggestion
2. It may fail if the instance hasn't set `binlog_format = ROW`.
```

5. Go back to Bytebase **SQL Editor**, and switch to **Admin Mode**.
![bb-sql-editor-admin](/static/blog/database-change-management-with-amazon-aurora/bb-sql-editor-admin.webp)

6. According to [Amazon documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/mysql-stored-proc-configuring.html#mysql_rds_show_configuration). Type as following to set the binlog retention hours to 24.
```SQL
call mysql.rds_set_configuration('binlog retention hours', 24);
```

7. Type as following to check it's set successfully.
```SQL
CALL mysql.rds_show_configuration;
```
![bb-sql-editor-admin-show-config](/static/blog/database-change-management-with-amazon-aurora/bb-sql-editor-admin-show-config.webp)

8. Reboot the Aurora MySQL instance.
![aws-reboot](/static/blog/database-change-management-with-amazon-aurora/aws-reboot.webp)

9. Repeat the Step 6, but this time, we can see the **Preview rollback issue**. Click **Create**.
![bb-issue-before-rollback-preview](/static/blog/database-change-management-with-amazon-aurora/bb-issue-before-rollback-preview.webp)
![bb-issue-rollback-preview](/static/blog/database-change-management-with-amazon-aurora/bb-issue-rollback-preview.webp)

> 10.  There is an **Approve** there even it's `Test`  environment. Since the rollback issue is a special case.

Reviewer: ```suggestion
10.  Rollback always requires explicit approval.
```

![bb-issue-rollback-to-approve](/static/blog/database-change-management-with-amazon-aurora/bb-issue-rollback-to-approve.webp)

11.  Before approving rollback, let's go to **SQL Editor** and query. `Bella` is there.
![bb-sql-editor-query-2](/blog/database-change-management-with-amazon-aurora/bb-sql-editor-query-2.webp)

1.  Go back to the issue page and click **Approve**. The rollback SQL will execute.
![bb-issue-rollback-executed](/static/blog/database-change-management-with-amazon-aurora/bb-issue-rollback-executed.webp)

> 1.  Go to **SQL Editor** and query again. `Bella` is not there any longer - The rollback for data insert is done. You may rollback the rollback too, and yes, rollback the rollback for rollback... As long as it's within the binlog retention 24-hour.

Reviewer: ```suggestion
1.  Go to **SQL Editor** and query again. `Bella` is no longer there - The rollback is successful. You may rollback the rollback too, and yes, rollback the rollback for rollback... As long as the binlog is within the 24-hour retention period.
```
